Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -492,15 +493,16 @@ def create_proxysg_out_format(iocs: list, category_attribute: list, category_def
 def create_mwg_out_format(iocs: list, mwg_type: str) -> dict:
     formatted_indicators = []  # type:List
     for indicator in iocs:
-        value = "\"" + indicator.get('value') + "\""
-        sources = indicator.get('sourceBrands')
-        if sources:
-            sources_string = "\"" + ','.join(sources) + "\""
+        if indicator.get('value'):
```

Comment:
better to use `if not indicator.get('value'): continue` to minimize the area under the indent